modes/basic-test-mode/src/index.ts

```diff
@@ -77,21 +77,23 @@ function modeFactory() {
       ]);
 
       let unsubscribe;
+      toolbarService.setDefaultTool({
+        groupId: 'WindowLevel',
+        itemId: 'WindowLevel',
```

Comment:
what is itemId? why we are adding it here

Reply:
`itemId` is already specified in the [docs](https://docs.ohif.org/platform/services/data/ToolbarService#api). This is setting the tool definition that should be used as the default when a tool is clicked to deactivate that tool. `itemId` is simply part of a tool definition and thus needs to be here... not sure how else to explain it. Does it make sense?

Reply:
can we assume it is default to groupId? Is it possible for them to be different?